bundles/org.openhab.binding.webexteams/src/main/resources/OH-INF/i18n/webexteams.properties

```diff
@@ -0,0 +1,37 @@
+# binding
+binding.webexteams.name = Webex Teams Binding
+binding.webexteams.description = Send messages to Webex Teams using actions
+
+# thing types
+thing-type.webexteams.account.label = Webex Teams Account
+thing-type.webexteams.account.description = Webex Teams Account
+
+# thing type config description
+thing-type.config.webexteams.account.token.label = Authentication token
+thing-type.config.webexteams.account.token.description = Authentication token for your Webex Teams bot
+thing-type.config.webexteams.account.roomId.label = Default room id
+thing-type.config.webexteams.account.roomId.description = Id of the room the sendMessage action will use
+
+# channel types
+channel-type.webexteams.botname.label = Bot Name
+channel-type.webexteams.botname.description = The name of the Webex Teams bot associated with the token
+
+# actions
+
+sendMessageActionLabel = send a Message to the default room
+sendMessageActionDescription = send a message to the default room
```

Comment:
I don't thing there is a need to have "message" with a leading uppercase letter.

The practice in other bindings is the following:
For the label: send a message to the default room
For the description: **Sends** a message to the default room

Reply:
Makes sense... updated.